<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Agregar</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <widget class="QPlainTextEdit" name="plainTextEdit"/>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="particle_groupBox">
          <property name="title">
           <string>Particle</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="5" column="0">
            <widget class="QLabel" name="velocidad_label">
             <property name="text">
              <string>Velocidad:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="origen_x_label">
             <property name="text">
              <string>Origen x:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QSpinBox" name="id_spinBox"/>
           </item>
           <item row="4" column="1" colspan="2">
            <widget class="QSpinBox" name="destino_y_spinBox">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="7" column="1" colspan="2">
            <widget class="QSpinBox" name="green_spinBox">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item row="9" column="0" colspan="2">
            <widget class="QPushButton" name="agregar_inicio_pushButton">
             <property name="text">
              <string>Agregar inicio</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1" colspan="2">
            <widget class="QSpinBox" name="red_spinBox">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QSpinBox" name="origen_x_spinBox"/>
           </item>
           <item row="5" column="2">
            <widget class="QSpinBox" name="velocidad_spinBox"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="id_label">
             <property name="text">
              <string>ID:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="origen_y_label">
             <property name="text">
              <string>Origen y:</string>
             </property>
            </widget>
           </item>
           <item row="9" column="2">
            <widget class="QPushButton" name="agregar_final_pushButton">
             <property name="text">
              <string>Agregar final</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="red_label">
             <property name="text">
              <string>Red:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QSpinBox" name="origen_y_spinBox"/>
           </item>
           <item row="10" column="0" colspan="3">
            <widget class="QPushButton" name="mostrar_pushButton">
             <property name="text">
              <string>Mostrar</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="destino_y_label">
             <property name="text">
              <string>Destino y:</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="green_label">
             <property name="text">
              <string>Green:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="destino_x_label">
             <property name="text">
              <string>Destino x:</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1" colspan="2">
            <widget class="QSpinBox" name="blue_spinBox">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="blue_label">
             <property name="text">
              <string>Blue:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1" colspan="2">
            <widget class="QSpinBox" name="destino_x_spinBox">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tabla</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="2">
         <widget class="QPushButton" name="buscar_pushButton">
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="3">
         <widget class="QTableWidget" name="tabla"/>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="mostrar_tabla_pushButton">
          <property name="text">
           <string>Mostrar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="buscar_lineEdit">
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Identificador de partícula</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Gráfico</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0" colspan="2">
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
        <item row="8" column="0" colspan="2">
         <widget class="QPushButton" name="limpiar">
          <property name="text">
           <string>Limpiar</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QPushButton" name="dibujar">
          <property name="text">
           <string>Dibujar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Grafos</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="2">
           <widget class="QGraphicsView" name="graphicsView_2"/>
          </item>
          <item row="0" column="0">
           <widget class="QTextBrowser" name="textBrowser"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_grafo">
          <property name="text">
           <string>Obtener Grafos</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Puntos Aleatorios</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="3" column="0">
         <widget class="QPushButton" name="Dibujar_Al_pushButton">
          <property name="text">
           <string>Dibujar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QGraphicsView" name="graphicsView_3"/>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="prim_pushButton">
          <property name="text">
           <string>Prim</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QSpinBox" name="spinBox_puntos">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="Dijkstra_pushButton">
          <property name="text">
           <string>Dijkstra</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="kruskal_pushButton">
          <property name="text">
           <string>Kruskal</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="2">
         <widget class="QPushButton" name="puntos_cercanos_al_pushButton">
          <property name="text">
           <string>Puntos Cercanos</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QSlider" name="horizontalSlider">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>939</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuAlgoritmos">
    <property name="title">
     <string>Algoritmos</string>
    </property>
    <addaction name="actionOrdenar_por_Id"/>
    <addaction name="actionOrdenar_por_Distancia_Rev"/>
    <addaction name="actionOrdenar_por_Velocidad"/>
    <addaction name="separator"/>
    <addaction name="actionPuntos_Cercanos"/>
    <addaction name="separator"/>
    <addaction name="actionPrim"/>
    <addaction name="actionGraham"/>
   </widget>
   <widget class="QMenu" name="menuVer">
    <property name="title">
     <string>Ver</string>
    </property>
    <addaction name="actionPuntos"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuVer"/>
   <addaction name="menuAlgoritmos"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionPuntos_Cercanos">
   <property name="text">
    <string>Puntos Cercanos</string>
   </property>
  </action>
  <action name="actionPuntos">
   <property name="text">
    <string>Puntos</string>
   </property>
  </action>
  <action name="actionOrdenar_por_Id">
   <property name="text">
    <string>Ordenar por Id</string>
   </property>
  </action>
  <action name="actionOrdenar_por_Distancia_Rev">
   <property name="text">
    <string>Ordenar por Distancia Rev</string>
   </property>
  </action>
  <action name="actionOrdenar_por_Velocidad">
   <property name="text">
    <string>Ordenar por Velocidad</string>
   </property>
  </action>
  <action name="actionDijkstra">
   <property name="text">
    <string>Dijkstra</string>
   </property>
  </action>
  <action name="actionPrim">
   <property name="text">
    <string>Prim</string>
   </property>
  </action>
  <action name="actionKruskal">
   <property name="text">
    <string>Kruskal</string>
   </property>
  </action>
  <action name="actionGraham">
   <property name="text">
    <string>Graham</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>destino_x_spinBox</tabstop>
  <tabstop>destino_y_spinBox</tabstop>
  <tabstop>red_spinBox</tabstop>
  <tabstop>green_spinBox</tabstop>
  <tabstop>blue_spinBox</tabstop>
  <tabstop>agregar_inicio_pushButton</tabstop>
  <tabstop>agregar_final_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
